docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -28,3 +26,187 @@ Workflow, show Workflow History, and Signal Workflow).
 - [temporal server](/cli/server)
 - [temporal task-queue](/cli/task-queue/)
 - [temporal workflow](/cli/workflow/)
+
+## Installation
+
+Temporal CLI can be installed through several different methods. While most of them can be used across all operating systems, please note that Homebrew is macOS-exclusive.
+
+For more information, see our guide to [running a Development Cluster](/clusters/how-to-install-temporal-cli).
+
+### cURL
+
+Run the following command to install the latest version of Temporal CLI.
+
+`curl -sSf https://temporal.download/cli.sh | sh`
+
+### Homebrew
+
+Run the following command in a new terminal window to install CLI for macOS.
+
+`brew install temporal`
+
+### GitHub releases
+
+Download and extract the latest release from [GitHub releases](https://github.com/temporalio/cli/releases).
+
+### CDN
+
+To install the Temporal CLI from CDN:
+
+1. Select the binary with the corresponding platform and architecture of your system.
+2. Download the binary.
+3. Extract the downloaded archive.
+4. Add the `temporal` binary to your system PATH.
+
+## Starting the Temporal Server
```

Comment:
```suggestion
## Starting a local dev Cluster
```

You will want to update this in source obviously :)

Reply:
And I know that using Cluster will be controversial to @bergundy - but embracing consistency with our terms is pretty important - and this consistency needs to apply to the CLI repo.
Since the current owners of that repo are failing to commit even after disagreeing... then we need to update this for them.

Reply:
You'll want to update the server repo too:
https://github.com/temporalio/temporal#download-and-start-temporal-server-locally
And docker compose:
https://github.com/temporalio/docker-compose#temporal-server-docker-compose-files
And pretty much every other repository that mentions Temporal server.